builders/build/substrate-api/substrate.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Polkadot's Substrate in Moonbeam
+description: Discover Polkadot's Substrate developer framework and how it shapes blockchain development in both Moonbeam's networks and other parachains.
+
+---
+
+# Polkadot's Substrate on Moonbeam
+
+![Substrate banner](/images/builders/build/substrate-api/substrate/substrate-banner.png)
+
+## Introduction {: #introduction }
+ 
+Polkadot's Substrate is an open-source, modular SDK for building blockchains in Rust. It is developed by Parity Technologies, the same team behind Polkadot.  
+
+Substrate provides the foundational building blocks for creating custom, domain-specific blockchains through its framework FRAME. FRAME is designed to be flexible and forkless, allowing developers to choose the components that best suit their blockchain needs and perform runtime upgrades without requiring network forks like Ethereum. Most developers will gravitate towards Substrate with the intention to integrate with the Polkadot ecosystem, but its flexibility allows users to create both solo-chains and parachains.  
```

Comment:
"Polkadot vs Substrate" are keywords that some search for, so it'd be great to explicitly touch on the relationship between polkadot and substrate, which I think you do here, but just more explicitly state "Polkadot does this, Substrate does this"